scripts/install_environment.sh

```diff
@@ -256,6 +256,11 @@ function additional_configs() {
     sudo sed -ir 's/net.ipv6.conf.all.disable_ipv6[[:blank:]]*=[[:blank:]]*1/net.ipv6.conf.all.disable_ipv6 = 0/g' /etc/sysctl.conf
     sudo sed -i -e '/net.core.somaxconn/d' -e '$a net.core.somaxconn = 2000' /etc/sysctl.conf
     sudo sysctl --load
+
+    if [ ${IPXE_BOOT-false} = "true" ]; then
+        echo "Opening port 8500 for iPXE boot."
+        sudo firewall-cmd --zone=libvirt --add-port=8500
```

Comment:
In my env it says:
```
# sudo firewall-cmd --zone=libvirt --add-port=8500
Error: INVALID_PORT: bad port (most likely missing protocol), correct syntax is portid[-portid]/protocol
```

But ``8500/tcp`` works. Maybe some operating systems are handling it differently 
I guess CI doesn't catch it because ``IPXE_BOOT`` is not enabled by default

Reply:
Thank you for catching this! Modified it to the correct syntax